redbot/cogs/admin/admin.py

```diff
@@ -40,18 +40,16 @@
 
 
 class Admin:
+
     def __init__(self, config=Config):
-        self.conf = config.get_conf(self, 8237492837454039,
-                                    force_registration=True)
+        self.conf = config.get_conf(self, 8237492837454039, force_registration=True)
 
-        self.conf.register_global(
-            serverlocked=False
-        )
+        self.conf.register_global(serverlocked=False)
 
         self.conf.register_guild(
             announce_ignore=False,
             announce_channel=None,  # Integer ID
-            selfroles=[]  # List of integer ID's
+            selfroles=[],  # List of integer ID's
```

Comment:
Where did this even come from. Is that compliant?

Reply:
Yes, it's compliant and a style choice.